framework/src/systems/NonlinearSystemBase.C

```diff
@@ -3444,55 +3468,57 @@ NonlinearSystemBase::computeScaling()
   // Compute our scaling factors for the spatial field variables
   for (const auto & elem : *mesh().getActiveLocalElementRange())
     for (MooseIndex(field_variables) i = 0; i < field_variables.size(); ++i)
+      if (field_variable_autoscaled[i])
```

Comment:
Kinda hard to see...but the only change here is adding this new "if" statement. The rest is just formatting.